sys-info-base.ts

```diff
@@ -207,6 +197,7 @@ export class SysInfoBase implements ISysInfo {
 			}
 		} catch (e) {
 			// if we got an error, assume not working
+			this.$logger.trace(`Error while executing child process: ${e}`);
```

Comment:
You can change `child process` to `${cmd}`. Also I think if we log `${e}` here, in the logs we will see `[object object]` instead of the error. You can log `e.message`.